lib/index.js

```diff
@@ -526,15 +458,11 @@ module.exports = class Microinvoice {
    * @return string
    */
   prettyPrice(value) {
-    if (typeof value === "number") {
-      value = value.toFixed(2);
-    }
-
     if (this.options.data.invoice.currency) {
-      value = `${value} ${this.options.data.invoice.currency}`;
+      return `${this.options.data.invoice.currency}${Number(value).toFixed(2)}`;
     }
 
-    return value;
+    return Number(value).toFixed(2);
```

Comment:
So I am a little bit worried that there would be cases where `NaN` might come up here but I think we can probably punt on doing that work since we do have some pre-validation before calling this Invoice generator. 

However as a standalone project eventually we should just add in a validation layer before we call our transform functions.

Reply:
@WrongKey  Just for your information, no need to make any changes now.

Reply:
Got it, thanks